src/main/java/com/google/cloud/tools/appengine/api/deploy/AppYamlProjectStageConfiguration.java

```diff
@@ -22,16 +22,16 @@
 import java.util.List;
 import javax.annotation.Nullable;
 
-/** Configuration for {@link AppEngineAppYamlStaging#stageArchive(StageAppYamlConfiguration)}. */
-public class StageAppYamlConfiguration {
+/** Application stager for appengine-web.xml based applications before deployment. */
```

Comment:
appengine-web.xml --> app.yaml

But this looks same as the `AppYamlProjectStaging` Javadoc. Maybe a different Javadoc about stage arguments?